packages/aeproject-cli/aeproject-contracts/aeproject-contracts.js

```diff
@@ -30,27 +48,53 @@ const mainFlow = async (options) => {
     });
 };
 
-const checkGlobalModuleExistence = (stdout, moduleName) => {
+const checkGlobalModuleExistence = async (stdout, moduleName) => {
     const devDependenciesPath = stdout.split('\n')[0];
     nodeModulesPath = path.join(devDependenciesPath, contractsConstants.NODE_MODULES_LITERAL);
     const modulesContent = fs.readdirSync(nodeModulesPath);
-    return modulesContent.includes(moduleName);
+
+    let isContractsInstalled = false;
+    // Travis throw exception when there are NO installed packages
+    let tempCwd = process.cwd();
+    try {
+        let yarnNodeModulePath = await getYarnNodeModulePath();
+        process.chdir(yarnNodeModulePath);
+
+        let yarnLS = await exec('ls');
+        isContractsInstalled = yarnLS.stdout.split('\n').indexOf('contracts-aepp') >= 0;
+    } catch (error) {
+        // When there is no installed package/directory is missing/, tests on travis throw error
+    }
+
+    process.chdir(tempCwd);
+
+    return modulesContent.includes(moduleName) || isContractsInstalled;
 };
 
 const installRepo = async () => {
+    const currentPath = process.cwd();
     console.log('====== Installing Contracts web Aepp ======');
-    await exec(`npm install -g ${ contractsConstants.CONTRACTS_AEPP_GITHUB_URL }`);
+
+    await exec(`yarn global add ${ contractsConstants.CONTRACTS_AEPP_GITHUB_URL }`);
+    process.chdir(path.resolve(await getYarnNodeModulePath(), 'contracts-aepp'))
+    await exec('yarn');
+
+    process.chdir(currentPath);
 };
 
 const serveContractsAepp = async (options) => {
     console.log('====== Starting Contracts web Aepp ======');
-    const currentDir = process.cwd();
-    process.chdir(contractAeppProjectPath);
 
+    let yarnGlobalDir = await exec('yarn global dir');
+    let contractAeppPath = path.resolve(yarnGlobalDir.stdout.replace('\n', ''), './node_modules/contracts-aepp');
```

Comment:
are you sure that changing the users global path for yarn is a good practice?

Reply:
I get it, not changing it